json-logic/README.md

```diff
@@ -0,0 +1,62 @@
+# CovQuestions Draft
+
+This is the draft of a custom questionaire format for the CovQuestions project.
+
+## Requirements
```

Comment:
Do we need to add multilingual support as a requirement? It's not defined as a MUST HAVE, but the example defines languages (https://github.com/alexanderthieme/CovQuestions/pull/1/files#diff-c123f194c2bd3ccf0d850a67a72f8debR4)

Reply:
Totally!

We can either:
1) Create a copy of the questionaire to apply another language. 
2) Allow indirections to a language-specific resource files for all human-readable strings.

I've taken care to seperate logic and human readable text everywhere for this reason. It should be straight forward to add.

Reply:
I would prefer the second approach.